generators/server/templates/package.json.ejs

```diff
@@ -29,7 +29,8 @@
     <%_ otherModules.forEach(module => { _%>
     "<%= module.name %>": "<%= module.version %>",
     <%_ }); _%>
-    "generator-jhipster": "<%= jhipsterVersion %>"
+    "generator-jhipster": "<%= jhipsterVersion %>",
+    "@openapitools/openapi-generator-cli": "0.0.9-3.3.4"
```

Comment:
Better to put 0.0.9-4.0.2

See https://github.com/jhipster/generator-jhipster/pull/9925